modules/core/src/main/scala/org/scalasteward/core/vcs/data/NewPullRequestData.scala

```diff
@@ -64,19 +64,22 @@ object NewPullRequestData {
       migrationLabel.toList ++
       oldVersionLabel.toList
 
-    s"""|Updates $artifacts ${fromTo(update)}.
-        |${releaseNote(releaseRelatedUrls).getOrElse("")}
+    s"""|Update $artifacts ${fromTo(update)}.
+        |* ${releaseNote(releaseRelatedUrls).getOrElse("")}
         |
-        |I'll automatically update this PR to resolve conflicts as long as you don't change it yourself.
-        |
-        |If you'd like to skip this version, you can just close this PR. If you have any feedback, just mention me in the comments below.
-        |
-        |Configure Scala Steward for your repository with a [`${RepoConfigAlg.repoConfigBasename}`](https://github.com/scala-steward-org/scala-steward/blob/${org.scalasteward.core.BuildInfo.gitHeadCommit}/docs/repo-specific-configuration.md) file.
+        |> I'll automatically update this PR to resolve conflicts as long as you don't change it yourself.
+        |> To skip this version update, just close this PR. If you have any feedback,
+        |> just mention me in the comments below.
+        |> Configure Scala Steward for your repository with a
+        |> [`${RepoConfigAlg.repoConfigBasename}`](https://github.com/scala-steward-org/scala-steward/blob/${org.scalasteward.core.BuildInfo.gitHeadCommit}/docs/repo-specific-configuration.md) file.
```

Comment:
What is a motivation to use block quote ?

I think plain paragraph is better than block quote, since it is not a quote but comment from Scala Steward.

Reply:
motivation here is that when you plug-in the bot, or when the repo have often dep updates, this text becomes repeated, so it looks as a noise. What i had in mind to get quickly focused on the more relevant text/description of the PR

Reply:
Yes but it conveys that it's quoting something when it's not.

Maybe abuse an html tag, like `<samp>`:

<samp>
I'll automatically update this PR to resolve conflicts as long as you don't change it yourself.
To skip this version update, just close this PR. If you have any feedback,
just mention me in the comments below.
Configure Scala Steward for your repository with a
[`.scala-steward.conf`](https://github.com/scala-steward-org/scala-steward/blob/${org.scalasteward.core.BuildInfo.gitHeadCommit}/docs/repo-specific-configuration.md) file.</samp>


Or `<kbd>`:

<kbd>
I'll automatically update this PR to resolve conflicts as long as you don't change it yourself.
To skip this version update, just close this PR. If you have any feedback,
just mention me in the comments below.
Configure Scala Steward for your repository with a
[`.scala-steward.conf`](https://github.com/scala-steward-org/scala-steward/blob/${org.scalasteward.core.BuildInfo.gitHeadCommit}/docs/repo-specific-configuration.md) file.
</kbd>

Reply:
yeah, abuse of "quote" functionality, but it grays out text, that's why used it. I'm more inclining to `<kbd>`, post here your thoughts, will update after